src/textinput/example/index.ts

```diff
@@ -39,8 +39,9 @@ export class App extends WidgetBase<WidgetProperties> {
 					type: 'text',
 					placeholder: 'Hello, World',
 					value: this._value1,
-					onChange: (event: TypedTargetEvent<HTMLInputElement>) => {
-						this._value1 = event.target.value;
+					onChange: (event: Event) => {
+						const target: HTMLInputElement = event.target as any;
```

Comment:
This differs from the `as HTMLInputElement` we seem to be preferring elsewhere in the code

Reply:
See above.